string name
---
robocup_msgs/Node node
